gef-ui/server/services/api.js

```diff
@@ -366,6 +366,33 @@ const getAmendment = async ({ facilityId, amendmentId, userToken }) => {
   }
 };
 
+/**
+ * @param {Object} param
+ * @param {string} param.facilityId
+ * @param {string} param.dealId
+ * @param {import('@ukef/dtfs2-common').PortalFacilityAmendmentUserValues} param.payload
+ * @param {string} param.userToken
+ * @returns {Promise<(import('@ukef/dtfs2-common').PortalFacilityAmendmentWithUkefId)>}>}
+ */
+const upsertAmendment = async ({ facilityId, dealId, payload, userToken }) => {
+  if (!isValidMongoId(facilityId)) {
+    throw new InvalidFacilityIdError(facilityId);
```

Comment:
Will these invoke `console.error`?

Reply:
not specifically a `console.error` although I believe this error will still show up in the console - I've added a `console.error` above this line :)